app/classification.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const teacherArray = ['taught','coached', 'trained', 'educated'];
+const studentArray = ['learned from'];
+const wroteArray = ['wrote','created', 'composed'];
+const playedArray = ['played','performed'];
+
+const dict = {teacher:teacherArray, student:studentArray, wrote:wroteArray, played:playedArray};
+
+module.exports.findRelationshipClass = function(word) {
+  for(let key in dict) {
+    let list = dict[key];
+    for(let w in list) {
+      if(list[w] === word) {
+        return key;
+      }
+    }
+  }
+  return null;
```

Comment:
yeah true! I got your point. May be we can associate some structure while returning from the function like :
```
{
"foundClassification": "true",
"relation" : "The_Relation"
}
if not found : 
{
"foundClassification": "false",
"relation" : "The_Word"
}
```
But the problem will arise when we will be storing it to Db, In the store to Db layer we have to check whether to store unclassified or not. Or we store everything ? Or we store unclassified separately ?

And also seems there will be another issue if we pass the word as 'taught by' and other as 'taught to', we will not be able to figure out whether it is entity 1 a teacher or its entity 2.  Or for the time being we can leave that and simply return teacher ? Or I think we will not classify it so simply return whatever the word is ?